bootstrap.py

```diff
@@ -132,7 +132,7 @@ def update(operation, verbose=None, upgrade=False, offline=False, optional_requi
     # We must install wheel first to eliminate a bunch of scary looking
     # errors at first install.
     # TODO Look towards fixing the packaging so that it works with 0.31
-    pip('install', ['wheel==0.30'], verbose, True, offline=offline)
```

Comment:
There is an issue with anything above 30.  By changing the base container this issue goes away because all of the dependencies are in the base for x86_64 architectures anyways